### ISSUE TITLE
Dataloader taking time

### ISSUE THREAD

**Issue by rahulsingh701** [milesial/Pytorch-UNet#317]
Hi, I am currently facing an issue where the dataloader is taking time to load the data. I am using the dfdc dataset. Can you suggest some alternatives to solve this issue?

**Comment by rahulsingh701** [comment]
<img width="843" alt="issue" src="https://user-images.githubusercontent.com/52104109/141257845-ce250443-b724-4338-8f54-fd2e6ee27ee9.png">
This is what I mean

**Comment by rahulsingh701** [comment]
It gets stuck for a very long time

**Comment by milesial** [comment]
If you know the pattern of filenames for images and masks in your dataset, you should change these lines https://github.com/milesial/Pytorch-UNet/blob/8f317cb13c17ef25a86b25a0c24390e04cd4db82/utils/data_loading.py#L58-L64 to to use `glob` anymore, but directly infer the filenames from the id.

**Comment by rahulsingh701** [comment]
Thanks @milesial. Issue got resolved. Its training fast now.
